### PR TITLE
chore(release): release v0.5.0

### PR DESCRIPTION
**Pull request by b5**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+<a name="v0.5.0"></a>
+# [v0.5.0](https://github.com/qri-io/qfs/compare/v0.4.2...v0.5.0) (2020-06-29)
+
+This release is jumping to v0.5.0 to skip a bad version (v0.4.2 was publishe in error, don't use!)
```
Comment (Arqu) [inline CHANGELOG.md:4]: publishe > published

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+v0.5.0 overhauls a bunch of APIS and introduces all sorts of breaking changes.
```
Comment (Arqu) [inline CHANGELOG.md:6]: APIs

**Review by Arqu — COMMENTED** [review]
Generally this is very coarse compared to what really changed. On the other hand `qfs` is super specific to us and doubt any real public use, so except the typos, works for me.

**Comment by b5** [comment]
yeah, agree this CHANGELOG is inadequate. I'd love to go back & set the record straight one day, but the bad version publication is messing up our `update-changelog` makefile routine, and I'm a little too pressed for time to fix. Like you said, this is a low level, low-outside-utility package, which leaves me less concerned

**Review by Arqu — APPROVED** [review]
🚀

**Pull request merged**